app/controllers/api/items_controller.rb

```diff
@@ -0,0 +1,74 @@
+module API
+  class ItemsController < ApplicationController
+    before_filter :validate_item, only: [:create, :update]
```

Comment:
I haven't run into this myself, @inukshuk.  Do you have any opinions about this, @jcoyne?